### PR TITLE
Gulp-Nodemon added in project

### PR DESCRIPTION
**Pull request by ahsankhan1911**
For enhancing the development environment  I have added gulp-nodemon in our project.

### CODE REVIEW

**Comment by khurrumqureshi** [comment]
@ahsankhan1911 why this branch contains work from TP-2. As I told you every branch should be independent and it will only contain code associated to particular task.

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let nodemon =  require('gulp-nodemon');
+const gulp = require('gulp');
+
+
+gulp.task('default', function () {
+  nodemon({
+    script: 'index.js'
+}),
+console.log("In gulpfile.js");
```
Comment (khurrumqureshi) [inline gulpfile.js]: remove console.log

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+let nodemon =  require('gulp-nodemon');
+const gulp = require('gulp');
+
+
+gulp.task('default', function () {
+  nodemon({
+    script: 'index.js'
```
Comment (khurrumqureshi) [inline gulpfile.js]: it should ignore node_modules directory, and only restart for js files only

**Comment by ahsankhan1911** [comment]
Should I remove the api directory for this branch?

**Comment by khurrumqureshi** [comment]
@ahsankhan1911 Resolve conflicts

**Comment by ahsankhan1911** [comment]
@khurrumqureshi  I am not able to understand this conflict. Can you please guide

**Pull request merged**